.Jules/palette.md

```
@@ -36,3 +36,9 @@
 ## 2026-07-30 - 無効化状態の理由の明示
 **Learning:** 非同期処理中にインタラクティブな要素（ボタンなど）を無効にする際、単に disabled 属性に頼るだけではスクリーンリーダーなどの支援技術ユーザーにコンテキストが伝わらない。
 **Action:** 無効化する際は常に title および aria-label 属性を設定し、なぜ無効化されているのか（例：'Cannot cancel while sending'）をすべてのユーザーに明示的に説明する。
+## 2026-08-05 - 送信ボタン無効化時のフォーカス復元（Composer）
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)